test/test_utils.py

```diff
@@ -52,3 +53,27 @@ def test_calculate_net_price(self):
         self.assertEqual(utils.calculate_net_price(Decimal('0.03'), publisher_fee, steam_fee), Decimal('0.01'))
         self.assertEqual(utils.calculate_net_price(Decimal('0.12'), publisher_fee, steam_fee), Decimal('0.10'))
         self.assertEqual(utils.calculate_net_price(Decimal('115'), publisher_fee, steam_fee), Decimal('100'))
+
+    def test_calculate_net_price_file(self):
+        steam_fee = Decimal('0.05')  # 5%
+        publisher_fee = Decimal('0.1')  # 10%
+
+        dirname = os.path.dirname(os.path.abspath(__file__))
+        csgo_market_selling_eff_taxes = dirname + "/data/csgo-market-selling-effective-taxes.txt"
+
+        with open(csgo_market_selling_eff_taxes, "r") as csgo_market_selling_eff_taxes_file:
+            lines = [line.rstrip() for line in csgo_market_selling_eff_taxes_file]
+
+        lines.pop(0)
+
```

Comment:
I think you can either drop the empty lines or convert code separating those lines into some function